vue3/src/main.js

```diff
@@ -0,0 +1,18 @@
+import { createApp } from 'vue';
+// import {
+//   applyPolyfills,
+//   defineCustomElements,
+// } from '@cdssnc/gcds-components-vue/loader';
+import { GcdsComponents } from '@cdssnc/gcds-components-vue';
+
+import '@cdssnc/gcds-components-vue/gcds.css';
+import './style.css';
+import App from './App.vue';
+
+// applyPolyfills().then(() => {
```

Comment:
Can we remove the comments from this file?

Reply:
+1